draft-ietf-ccwg-bbr.md

```diff
@@ -754,6 +754,18 @@ protocols, and these implementations have been used in production
 for a large volume of Internet traffic. An open source implementation of
 BBR is also available for DCCP {{RFC4340}}  {{draft-romo-iccrg-ccid5}}.
 
+## ECN
+
+This experimental version of BBR does not specify a specific response to
+either Classic {{RFC3168}} or L4S {{RFC9330}} style ECN. However, if
```

Comment:
```suggestion
Classic {{RFC3168}}, Alternative Backoff with ECN (ABE) {{RFC8511}} or L4S {{RFC9330}} style ECN. However, if
```

It seems strange to describe this as if Classic or L4S were the only two possible options. Really, we have:
- Classic, PS, although updated by RFC 8311, PS
- Experimental: ABE or L4S.